includes/rest-api.php

```diff
@@ -263,7 +264,8 @@ function register_endpoints() {
 		'distributor_original_site_url',
 		array(
 			'get_callback'    => function( $post_array ) {
-				return esc_url_raw( get_post_meta( $post_array['id'], 'dt_original_site_url', true ) );
+				$site_url = get_post_meta( $post_array['id'], 'dt_original_site_url', true );
+				return empty( $site_url ) ? home_url() : esc_html( $site_url );
```

Comment:
Seems like we should probably have the escaping here be `esc_url_raw` instead of `esc_html`

Reply:
Yeah, it's my mistake. I fixed it in https://github.com/10up/distributor/pull/767/commits/1e49b581bfaa0701f0885a36a7f1d0eed96b5441